torchvision/prototype/transforms/_augment.py

```diff
@@ -105,9 +105,7 @@ def __init__(self, *, alpha: float, p: float = 0.5) -> None:
 
     def forward(self, *inpts: Any) -> Any:
         sample = inpts if len(inpts) > 1 else inpts[0]
-        if not (
-            has_any(sample, features.Image, PIL.Image.Image, is_simple_tensor) and has_any(sample, features.OneHotLabel)
-        ):
+        if not (has_any(sample, features.Image, is_simple_tensor) and has_any(sample, features.OneHotLabel)):
```

Comment:
To understand better, PIL Image was not and was bypassed previously and now we have an explicit error on PIL Image input ?